### ISSUE TITLE
Need consistent way for capturing approvals as part of Service Partner Library creation process

### ISSUE THREAD

**Issue by MrJustinB** [Azure/azure-sdk-tools#4278]

**Comment by MrJustinB** [comment]
This includes API Stewardship Board, Azure SDK Architecture Board, Azure.core, and ARM approvals

**Comment by josefree** [comment]
ARM uses the PR label "ARMSignedOff"
API stewardship agree to use the PR label "APIStewardshipBoard-SignedOff"

**Comment by MrJustinB** [comment]
Is this signoff reflected in the release planner?  That is, is the release planner monitoring the PR to reflect the signofff?

**Comment by josefree** [comment]
Yes. in API spec readiness, Signoff label is required for 'next'.
And in SDK release, a spec pull request with signoff label is one of prerequisite checks.

**Comment by josefree** [comment]
The missing part is ArchBoard review signoff.  Not sure what's the exit signal.

**Comment by maririos** [comment]
> The missing part is ArchBoard review signoff. Not sure what's the exit signal.

APIView has this information. It has a REST API to tell the status of a review

**Comment by maririos** [comment]
@dw511214992 I believe we already have this logic in the SDK Release app so we can close this issue. Right?

**Comment by maririos** [comment]
> @maririos Is this done?

Unclear. We will need to check the code and see from there what we find :)

**Comment by maririos** [comment]
With the new update to the SDK tools, we are not looking at the state of the reviews.
Leaving this issue as an automation improvement
